lib/ansible/modules/utilities/logic/wait_for.py

```diff
@@ -369,13 +374,15 @@ def _create_connection(host, port, connect_timeout):
     Returns:
         Socket object
     """
-    if sys.version_info < (2, 6):
```

Comment:
I don't think we should be removing older python support just yet.

https://groups.google.com/forum/#!topic/ansible-devel/BvhGM3wjH54

@abadger or @bcoca ?